lib/msf/ui/console/command_dispatcher/common.rb

```diff
@@ -158,6 +158,31 @@ def index_from_list(list, index, &block)
     yield list[idx]
   end
 
+  # Trims starting `.`, `./` `/`, `+path_head+/`, & `/+path_head+/` from +path+. Also trims trailing `.+extension+`
+  # from +path+, and any possible combination of misspellings of +extension+.
+  #
+  # @param path [String] The path to be trimmed
+  # @param path_head [String] The top-level directory that should be removed from the path
+  # @param extension [String] File extension to be trimmed from +path+. `.` is automatically included. Defaults to `rb`.
+  # @return [String] Altered +path+. Will return unaltered +path+ if regex constructed with +path_head+ & +path+ is not detected
+  def trim_path(path, path_head, extension="rb")
+    regexp = %r{
+        (
+          ^\.?                # Dot at beginning of path
+          /?                  # Slash at beginning of path
+          (?:#{path_head}/)?  # top level directory (slash prepending TLD is optional)
```

Comment:
I've never seen TLD used as an acronym for top level directory, I would always read it as Top Level Domain, maybe we can get away without using it

Reply:
Sure, comments are meant to be clear so we may as well pull any ambiguous acronyms